src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -506,6 +510,36 @@ public void delete() throws IOException {
     }
 
     /**
+     * Sort orders for listing forks
+     */
+    public static enum Sort { NEWEST, OLDEST, STARGAZERS }
```

Comment:
Everyone adding code in their own particular coding style also makes this library hard to read and understand. Also, this is not some cool new feature that needed to be implemented, I just added one missing api method (out of, I don't know, a hundred?), that for some reason has been overlooked in the past.

Implementing this in a "new way" by someone like me who doesn't know this library at all is way more error prone, as I don't know the particulars of the currently chosen approach.

Also now, if someone wants to clean up this code, at least everything looks the same, which imho makes cleaning up much easier than if everything looks different.